zombies.js

```diff
@@ -240,14 +363,29 @@
  * @param {number} strength         The zombie's strength.
  * @param {number} speed            The zombie's speed.
  */
+class FastZombie extends Zombie{
+  constructor(health, strength, speed){
+    super();
```

Comment:
Since the Super class for FastZombie takes health, strength, speed as arguments. you need to pass them to super(), not set them in the child class.